### PR TITLE
Add changelog for release-1.4.1

### PR DESCRIPTION
**Pull request by pankajastro**
Add changelog for release-1.4.1

### CODE REVIEW

**Comment by kaxil** [comment]
@pankajastro - I would like to get https://github.com/astronomer/astro-sdk/pull/1647 in too

File: python-sdk/docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+- Add the documentation and example DAG for Azure blob storage [#1598](https://github.com/astronomer/astro-sdk/pull/1598)
+- Add step to generate constraints in Python SDK release process [#1474](https://github.com/astronomer/astro-sdk/issues/1474)
+- Fix dead link in documentation [#1596](https://github.com/astronomer/astro-sdk/pull/1596)
+- Update README with newly supported location and db [#1596](https://github.com/astronomer/astro-sdk/pull/1579)
+
+### Misc
+- install `google-cloud-sdk-gke-gcloud-auth-plugin` in benchmark ci job [#1557](https://github.com/astronomer/astro-sdk/issues/1557)
+- Pin sphinx-autoapi==2.0.0 version for docs build [#1609](https://github.com/astronomer/astro-sdk/pull/1609)
+
+
```
Comment (kaxil) [inline python-sdk/docs/CHANGELOG.md:15]:
```suggestion
## 1.4.1

## Bug fixes
- Support "s3" conn type for S3Location [#1647](https://github.com/astronomer/astro-sdk/pull/1647)

### Docs
- Add the documentation and example DAG for Azure blob storage [#1598](https://github.com/astronomer/astro-sdk/pull/1598)
- Fix dead link in documentation [#1596](https://github.com/astronomer/astro-sdk/pull/1596)
- Update README with newly supported location and db [#1596](https://github.com/astronomer/astro-sdk/pull/1579)
- Update configuration reference for XCom [#1646](https://github.com/astronomer/astro-sdk/pull/1646)
- Add step to generate constraints in Python SDK release process [#1474](https://github.com/astronomer/astro-sdk/issues/1474)


### Misc
- install `google-cloud-sdk-gke-gcloud-auth-plugin` in benchmark ci job [#1557](https://github.com/astronomer/astro-sdk/issues/1557)
- Pin `sphinx-autoapi==2.0.0` version for docs build [#1609](https://github.com/astronomer/astro-sdk/pull/1609)
```

File: python-sdk/docs/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## 1.4.1
+
+### Feature:
+- Implement `check_table` Operator to validate data quality at table level [#1239](https://github.com/astronomer/astro-sdk/pull/1239)
```
Comment (sunank200) [inline python-sdk/docs/CHANGELOG.md:6]: Should we also add a PR for the documentation here for users?

File: python-sdk/docs/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+- Add `check_column` Operator to validate data quality for columns in a table/dataframe [#1239](https://github.com/astronomer/astro-sdk/pull/1239)
```
Comment (sunank200) [inline python-sdk/docs/CHANGELOG.md:7]: Should we also add a PR for the documentation here for users?

File: python-sdk/docs/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,27 @@
...
+
+## Bug fixes
+- Support "s3" conn type for S3Location [#1647](https://github.com/astronomer/astro-sdk/pull/1647)
```
Comment (sunank200) [inline python-sdk/docs/CHANGELOG.md:10]: This is not a bug fix. Is it? This was added to the airflow and we support it too right?

File: python-sdk/docs/CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,27 @@
...
+
+### Docs
+- Add the documentation and example DAG for Azure blob storage [#1598](https://github.com/astronomer/astro-sdk/pull/1598)
+- Fix dead link in documentation [#1596](https://github.com/astronomer/astro-sdk/pull/1596)
+- Update README with newly supported location and db [#1596](https://github.com/astronomer/astro-sdk/pull/1579)
```
Comment (sunank200) [inline python-sdk/docs/CHANGELOG.md:15]:
```suggestion
- Update README with newly supported location and database [#1596](https://github.com/astronomer/astro-sdk/pull/1579)
```

File: python-sdk/docs/CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,27 @@
...
+- Update configuration reference for XCom [#1646](https://github.com/astronomer/astro-sdk/pull/1646)
+- Add step to generate constraints in Python SDK release process [#1474](https://github.com/astronomer/astro-sdk/issues/1474)
+- Add document to showcase the use of `check_table` and `check_column` operators [#1631](https://github.com/astronomer/astro-sdk/pull/1631)
+
+### Misc
+- install `google-cloud-sdk-gke-gcloud-auth-plugin` in benchmark ci job [#1557](https://github.com/astronomer/astro-sdk/issues/1557)
```
Comment (sunank200) [inline python-sdk/docs/CHANGELOG.md:21]:
```suggestion
- Install `google-cloud-sdk-gke-gcloud-auth-plugin` in benchmark CI job [#1557](https://github.com/astronomer/astro-sdk/issues/1557)
```

File: python-sdk/docs/CHANGELOG.md (hunk 4 above)
Comment (pankajastro) [reply python-sdk/docs/CHANGELOG.md:10]: This bug, this got removed from airflow but wanted to support for older version

File: python-sdk/docs/CHANGELOG.md (hunk 2 above)
Comment (pankajastro) [reply python-sdk/docs/CHANGELOG.md:6]: We have added that I docs section

File: python-sdk/docs/CHANGELOG.md (hunk 3 above)
Comment (pankajastro) [reply python-sdk/docs/CHANGELOG.md:7]: same here

Comment (utkarsharma2) [reply python-sdk/docs/CHANGELOG.md:7]: We added in the doc section,

**Review by utkarsharma2 — APPROVED** [review]

**Pull request merged**